compose-builder/common.env

```diff
@@ -18,12 +18,11 @@
 
 EDGEX_SECURITY_SECRET_STORE=false
 REGISTRY_HOST=edgex-core-consul
-CLIENTS_COREDATA_HOST=edgex-core-data
-CLIENTS_DATA_HOST=edgex-core-data
-CLIENTS_NOTIFICATIONS_HOST=edgex-support-notifications
-CLIENTS_METADATA_HOST=edgex-core-metadata
-CLIENTS_COMMAND_HOST=edgex-core-command
-CLIENTS_SCHEDULER_HOST=edgex-support-scheduler
```

Comment:
The old versions can not be removed until the device and app services have all been updated to uses service keys. Please add back with a TODO to remove later.

Reply:
We also have an issue with removing these since Proxy Setup uses these. @hutchic,  @jim-wang-intel , @bnevis-i , we need to discuss this in Security WG.

Reply:
so we can modify the toml in proxy-setup accordingly, and it should work?

Reply:
https://github.com/edgexfoundry/edgex-go/blob/master/cmd/security-proxy-setup/res/configuration.toml#L57-L91

Reply:
> so we can modify the toml in proxy-setup accordingly, and it should work?

Not sure we can as the names are used in the proxy path, correct? 
Do we want the service key's to be the name in the proxy path?

Reply:
the servicename is from `configuration.Clients` from bootstrapConfig.ClientInfo, so it is used in proxy path.  The question is like you mentioned, whether we want to use **service keys** as the proxy path, worth discussion in the meeting.

Reply:
Update from today's Security WG meeting: We will rework proxy Setup's config to not use repurposed Client configuration and create it's own proxy specific configuration struct. But until then the old overrides need to stay in place.

Reply:
Added old version env back.